core/esmf-aspect-model-document-generators/src/test/java/org/eclipse/esmf/aspectmodel/generator/sql/databricks/AspectModelDatabricksDenormalizedSqlVisitorTest.java

```diff
@@ -209,7 +213,7 @@ void testAspectWithEntityWithNestedEntityListProperty() {
       assertThat( sql( TestAspect.ASPECT_WITH_ENTITY_WITH_NESTED_ENTITY_LIST_PROPERTY ) ).isEqualTo( """
             CREATE TABLE IF NOT EXISTS aspect_with_entity_with_nested_entity_list_property (
               test_property__code SMALLINT NOT NULL,
-              test_property__test_list ARRAY<STRUCT<nested_entity_property: STRING NOT NULL>> NOT NULL
+              nested_entity_property STRING NOT NULL
```

Comment:
Like in my other comment, the structure still needs to be present, so this should be:
`test_property__nested_entity_property STRING NOT NULL`